src/services/business/fetch-business-email-change-service.js

```diff
@@ -0,0 +1,17 @@
+import { fetchBusinessDetailsService } from './fetch-business-details-service.js'
```

Comment:
Imports should come after the module declaration

Reply:
Module declaration removed.